src/HashMapCreate.java

```diff
@@ -0,0 +1,180 @@
+// Operation:            put         get          remove      findPrev
+// Time Complexity:      O(n)        O(n)          O(n)         O(n)
+// Space Complexity:     O(n)        O(n)          O(n)         O(n)
```

Comment:
Good! Please try to implement the O(1) solution as the important feature of HashMap is O(1) access time.